modules/lx-instance/watchmaker-lx-instance.template.cfn.yaml

```diff
@@ -538,22 +538,15 @@ Resources:
               - !Ref 'AWS::NoValue'
         cw-agent-install:
           commands:
-            01-get-cloudwatch-agent:
+            01-install-cloudwatch-agent:
               command: !Sub >-
-                install -Dbm 700 -o root -g root /dev/null
-                /etc/cfn/scripts/AmazonCloudWatchAgent.zip &&
-                aws s3 cp ${CloudWatchAgentUrl}
-                /etc/cfn/scripts/AmazonCloudWatchAgent.zip
-            02-extract-cloudwatch-agent:
-              command: >-
-                yum -y install unzip &&
-                unzip /etc/cfn/scripts/AmazonCloudWatchAgent.zip -d /etc/cfn/scripts/aws-cw-agent
+                aws s3 cp ${CloudWatchAgentUrl} /etc/cfn/scripts/amazon-cloudwatch-agent.rpm &&
+                rpm --quiet -U /etc/cfn/scripts/amazon-cloudwatch-agent.rpm
```

Comment:
Unless it causes problems, it makes debugging install issues easier to have `-v` instead of `--quiet`.

Reply:
I'll make the switch.  Still works fine with `-v`.